ddmrp/models/stock_buffer.py

```diff
@@ -1240,19 +1202,19 @@ def _compute_product_vendor_code(self):
         comodel_name="mrp.production",
         inverse_name="buffer_id",
     )
-    ddmrp_chart = fields.Text(
+    ddmrp_chart = fields.Json(
         string="DDMRP Chart",
         compute=_compute_ddmrp_chart_planning,
     )
-    ddmrp_chart_execution = fields.Text(
-        string="DDMRP Execution Chart", compute=_compute_ddmrp_chart_execution
+    ddmrp_chart_execution = fields.Json(
+        string="DDMRP Execution Chart", compute=_compute_ddmrp_chart_planning
```

Comment:
ddmrp_chart_execution vs _compute_ddmrp_chart_planning, it doesn't look correct

Reply:
Resolved!